### PR TITLE
Add pending models

### PR DESCRIPTION
**Pull request by isaintmartin**
- Add coffee model

### CODE REVIEW

File: library/src/main/java/com/mooveit/library/providers/CoffeeProviderImpl.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.mooveit.library.providers
+
+import com.mooveit.library.Fakeit.Companion.fakeit
+import com.mooveit.library.providers.base.BaseProvider
+import com.mooveit.library.providers.definition.CoffeeProvider
+
+class CoffeeProviderImpl : BaseProvider(), CoffeeProvider {
```
Comment (Jeasmine) [inline library/src/main/java/com/mooveit/library/providers/CoffeeProviderImpl.kt]: Agregate un enter 🙏

File: library/src/main/java/com/mooveit/library/providers/CoffeeProviderImpl.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    override fun blendName(): String = getValue("blend_name", { fakeit!!.fetch("coffee.blend_name") })
+
+    override fun origin(): String {
+        val country = getValue("country", { fakeit!!.fetch("coffee.country") })
+        val country_formatted = country.replace(" ", "_").toLowerCase()
+        return country + ", " + getValue("country", { fakeit!!.fetch("coffee.regions.$country_formatted") })
```
Comment (Jeasmine) [inline library/src/main/java/com/mooveit/library/providers/CoffeeProviderImpl.kt]: It should be region instead of country

Comment (isaintmartin) [reply library/src/main/java/com/mooveit/library/providers/CoffeeProviderImpl.kt]: Done

File: library/src/main/java/com/mooveit/library/providers/CoffeeProviderImpl.kt (hunk 1 above)
Comment (isaintmartin) [reply library/src/main/java/com/mooveit/library/providers/CoffeeProviderImpl.kt]: Done

**Pull request merged**